code/modules/spells/general/mark_recall.dm

```diff
@@ -36,7 +36,7 @@
 	var/turf/T = get_turf(target)
 	if(!T)
 		return 0
-	user.forceMove(T)
+	do_teleport(user,T)
```

Comment:
Nope, wizard powers should not be forced to suffer the same restrictions as telescience.

Reply:
The wizard should not be able to teleport into the vault tbh.

Reply:
It is magic, not bluespace.